cfme/tests/configure/test_visual_cloud.py

```diff
@@ -38,31 +38,36 @@
 }
 
 
+@pytest.fixture(scope="module")
+def visual():
+    return Visual
```

Comment:
**required** this bit is missing a instantiation

Reply:
nice catch ;)